build/wikibase/README.md

```diff
@@ -43,6 +43,7 @@ Variables in **bold** are required on first launch without `LocalSettings.php` i
 
 | Variable                     | Default    | Description                                                                                                                                                                                                  |
 | ---------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **`IS_JOBRUNNER`**              | false  | If set to true the container will only run `maintenance/runJobs.php` instead of MediaWiki                                                                                                                                       |
```

Comment:
This is not a breaking change, is it? As I see it, the image can still be used with the old healthcheck style and be set to "job runner mode" by manually choosing ` /jobrunner-entrypoint.sh`. Is that correct?

Reply:
I'm pretty sure that is true, but I would think of this previous was as deprecated as of this update.